src/plugins/initiatives/initiativesMap.js

```diff
@@ -30,3 +33,90 @@ export function fromJSON(j: Compatible<any>): InitiativesMap {
 export function toJSON(m: InitiativesMap): Compatible<InitiativesMap> {
   return toCompat(COMPAT_INFO, m);
 }
+
+/**
+ * Creates Initiatives from a InitiativesMap.
+ */
+export function mapToInitiatives(
+  initiativesMap: InitiativesMap
+): $ReadOnlyArray<Initiative> {
+  const initiatives = [];
+  for (const entryKey in initiativesMap) {
+    const addr = _trackerAddress(entryKey);
+    initiatives.push({
+      ...initiativesMap[entryKey],
+      tracker: addr,
+    });
+  }
+  return initiatives;
+}
+
+export function _trackerAddress(entryKey: string): NodeAddressT {
+  return NodeAddress.append(trackerPrefix, entryKey);
+}
+
+/**
+ * Creates an InitiativesMap from Initiatives. Warning: this is a lossy
+ * operation, as the original tracker address will be dropped!
```

Comment:
I don't understand what this means. What original tracker address is being dropped? It appears that what you really mean is that this method (which I propose calling `createInitiativesMap`) does not faithfully include the input initiatives, but rather re-writes their keys in attempt to avoid collissions. is this correct?

Reply:
That's correct.

The tracker is the node address of the source which produced the initiative.
Such as a Discourse topic.
https://github.com/sourcecred/sourcecred/blob/c9958e346ee789f41e1e406acc864c241245e70e/src/plugins/initiatives/initiative.js#L25

The node address of the initiative node itself is a prefix + the tracker
https://github.com/sourcecred/sourcecred/blob/c9958e346ee789f41e1e406acc864c241245e70e/src/plugins/initiatives/createGraph.js#L22-L27